redfish/session.go

```diff
@@ -103,7 +102,7 @@ func CreateSession(c common.Client, uri, username, password string) (auth *AuthT
 
 // DeleteSession deletes a session using the location as argument
 func DeleteSession(c common.Client, sessionURL string) (err error) {
-	resp, err := c.Delete(sessionURL)
+	resp, err := c.Delete("/redfish/v1/SessionService/Sessions/" + sessionURL)
```

Comment:
Can you confirm - is the issue here from `client.go` where `DeleteSession` is being called, that it is only passing through the session ID rather than the full URL path for the session? I haven't been able to test with a real system.

If that is the case, please rename the parameter variable name to `sessionID` to make it clear what is expected to be passed in to this call, as well as in `serviceroot.go`.

Thanks!